lib/ui/addon/components/error-page.js

```diff
@@ -3,12 +3,5 @@ import layout from '../templates/components/error-page';
 
 export default Component.extend({
   layout,
-
-  didInsertElement() {
```

Comment:
Just use CSS for this, duh.

Reply:
Can you explain more about this?

Reply:
I just use CSS to animate Tomster in place of manual DOM stuff.

Reply:
Gotcha, sounds good 👍